build/istanbul-config.js

```diff
@@ -1,24 +1,20 @@
 'use strict';
 
-var path = require('path');
+var unitTestCoverageDirectory = './coverage/unit';
+var unitTestCoverageReportHtmlFile = './coverage/unit/index.html';
 
```

Comment:
why remove the path.resolve on the coverage directory and the html file? I can't remember the specifics, but I had to add these to resolve an issue